Dockerfile.flodgatt

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:18.04
+
+# Create the mastodon user
+ARG UID=991
+ARG GID=991
+RUN apt update && \
+	echo "Etc/UTC" > /etc/localtime && \
+	apt install -y whois wget && \
```

Comment:
```suggestion
	apt install --no-install-recommends -y whois wget && \
```

Reply:
I've taken this advice, and made some changes.